services/github/github-manifest.service.js

```diff
@@ -25,8 +25,7 @@ class GithubManifestVersion extends ConditionalGithubAuthService {
   static get route() {
     return {
       base: 'github/manifest-json/v',
-      format: '([^/]+)/([^/]+)/?([^/]+)?',
-      capture: ['user', 'repo', 'branch'],
+      pattern: ':user/:repo/:branch?',
```

Comment:
Sidenote, I'm seeing an `invalid response data` badge message for the example badge used here. Seeing this in prod too so not related to this PR.

https://img.shields.io/github/manifest-json/v/RedSparr0w/IndieGala-Helper.svg